recipes/default.rb

```diff
@@ -47,6 +47,12 @@
   cwd "#{node['timesync']['application_path']}/source"
 end
 
+bash 'create root user' do
+  code 'npm run create-user -- -u root -p #{secret_key['root_pass']}'
```

Comment:
`'root_pass'` inside a single quote string is a problem, both because it terminates the string quote, and because `#{}` only works within double quotes.